starter/spec.js

```diff
@@ -1,9 +1,195 @@
+            //Check if the Files page is displayed - verifying the page Url
+            browser.getCurrentUrl().then(function (text){
+                expect(text).toEqual(filesUrl);
+            });
+        });
+        it('should pop-up a new folder dialog box', function(){
+            //Clicks on "create new folder" icon and checks that a dialog box is displayed
+            createNewFolderIcon.click().then(function(){
+                browser.wait(EC.presenceOf(newFolderDialogBox), 5000).then(function(){
+                    expect((newFolderDialogBox).isDisplayed()).toBe(true);
+                })
+            })
+        });
+        it('should create a new folder', function () {
+            //Set the name for new folder
+            browser.wait(EC.elementToBeClickable((gitHubUsernameField), 5000)).then(function(){
+                gitHubUsernameField.click().then(function(){
+                    gitHubUsernameField.sendKeys(githubUsername);
+                });
+            });
+            //Create the folder and verify that the dialog box is not displayed anymore
+            createButton.click().then(function(){
+                browser.wait(EC.invisibilityOf(newFolderDialogBox), 5000);
+                expect((newFolderDialogBox).isPresent()).toBe(false);
+            })
+        });
+        it('should displayed my new abc folder', function(){
+            //Checks if the new folder is created
+            expect(newFolderName.isDisplayed()).toBe(true);
+        });
+
+        // it('should check for a duplicate message', function(){
+        //   // Click on "Create new folder" icon
+        //     createNewFolderIcon.click().then(function(){
+        //         browser.wait(EC.presenceOf(newFolderDialogBox), 5000).then(function(){
+        //             expect((newFolderDialogBox).isDisplayed()).toBe(true);
+        //         })
+        //     });
+        //     //Set the name as before
+        //     browser.wait(EC.elementToBeClickable((gitHubUsernameField), 5000)).then(function(){
+        //         gitHubUsernameField.click().then(function(){
+        //             gitHubUsernameField.sendKeys(githubUsername);
+        //         });
+        //     });
+        //     //Try to create the folder and verify that the dialog box is still displayed
+        //     createButton.click().then(function(){
+        //         expect((newFolderDialogBox).isPresent()).toBe(true);
+        //     });
+        //     //Verify that duplicate message error is displayed
+        //     expect(errorMessage.isDisplayed()).toBe(true);
+        // });
+
+        it('should delete my new folder', function(){
+            //Selects a specified folder and deletes it
+            newFolderName.click().then(function(){
+                threeDots.click().then(function(){
+                    expect(folderMenuOptions.isDisplayed()).toBe(true).then(function(){
+                        deleteFolderOptionButton.click();
```

Comment:
Can you please add the assertion for the deleted folder message that is displayed to the user.

Reply:
Hello, done. I've confirmed that the folder is deleted by checking that the confirmation message is displayed and that it contains the "deleted" keyword.